sdk/appplatform/arm-appplatform/CHANGELOG.md

```diff
@@ -226,24 +299,35 @@
   - Interface ManagedIdentityProperties has a new optional parameter userAssignedIdentities
   - Interface NetworkProfile has a new optional parameter ingressConfig
   - Interface NetworkProfile has a new optional parameter outboundType
-  - Class AppPlatformManagementClient has a new parameter apiPortalCustomDomains
-  - Class AppPlatformManagementClient has a new parameter apiPortals
-  - Class AppPlatformManagementClient has a new parameter gatewayCustomDomains
-  - Class AppPlatformManagementClient has a new parameter gatewayRouteConfigs
-  - Class AppPlatformManagementClient has a new parameter gateways
-  - Class AppPlatformManagementClient has a new parameter storages
   - Added Enum KnownApiPortalProvisioningState
+  - Added Enum KnownApmType
+  - Added Enum KnownApplicationAcceleratorProvisioningState
+  - Added Enum KnownApplicationLiveViewProvisioningState
   - Added Enum KnownBackendProtocol
   - Added Enum KnownCertificateResourceProvisioningState
   - Added Enum KnownCustomDomainResourceProvisioningState
+  - Added Enum KnownCustomizedAcceleratorProvisioningState
+  - Added Enum KnownCustomizedAcceleratorValidateResultState
+  - Added Enum KnownDevToolPortalFeatureState
+  - Added Enum KnownDevToolPortalProvisioningState
   - Added Enum KnownGatewayProvisioningState
   - Added Enum KnownGatewayRouteConfigProtocol
   - Added Enum KnownHttpSchemeType
   - Added Enum KnownPowerState
+  - Added Enum KnownPredefinedAcceleratorProvisioningState
+  - Added Enum KnownPredefinedAcceleratorState
   - Added Enum KnownProbeActionType
   - Added Enum KnownSessionAffinity
   - Added Enum KnownStorageType
   - Added Enum KnownType
+  - Enum KnownBindingType has a new value CACertificates
+  - Added function getContinuationToken
+
+**Breaking Changes**
+
+  - Interface DeploymentsListForClusterNextOptionalParams no longer has parameter version
+  - Interface DeploymentsListNextOptionalParams no longer has parameter version
+  - Type of parameter type of interface UserSourceInfo is changed from "UploadedUserSourceInfo" | "Jar" | "Source" | "NetCoreZip" | "BuildResult" to "UploadedUserSourceInfo" | "Jar" | "Source" | "NetCoreZip" | "BuildResult" | "Container"
```

Comment:
this breaking chagne occurred during the last release, but it was not detected at that time

Reply:
I don't think this is breaking. Could you move to non-breaking part?

Tool issue tracked: https://github.com/Azure/azure-sdk-tools/issues/5055